NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -162,15 +163,56 @@ public StateQuery()
                                     }
                                 }
 #pragma warning disable CS0618 // Type or member is obsolete
-                                arenastate.OrderedArenaInfos.AddRange(arenaInfos.OrderByDescending(a => a.Score).ThenBy(a => a.CombatPoint));
+                                arenastate.OrderedArenaInfos.AddRange(arenaInfos.OrderByDescending(a => a.Score)
+                                    .ThenBy(a => a.CombatPoint));
 #pragma warning restore CS0618 // Type or member is obsolete
                             }
                         }
+
                         return arenastate;
                     }
 
                     return null;
                 });
+            Field<NonNullGraphType<ListGraphType<ArenaInformationType>>>(
+                name: "arenaInformation",
```

Comment:
Since it returns a list, it seems better to use plural 🤔 

```suggestion
                name: "arenaInformations",
```

Reply:
Oops, you're right 👍🏻